lib/job_tomate/actions/jira_update_issue_assignee_and_roles_for_status.rb

```diff
@@ -116,10 +116,13 @@ def update_assignee_and_role(issue, user, role)
 
       def unassign(issue)
         update_issue(issue, fields: { assignee: nil })
+        issue.change_assignee_name(nil)
```

Comment:
I think I understand what you didn't ;) `issue` is a value-object. I think it should be immutable, it would make it clearer that this line is not useful. You're changing `issue` but it's only changing the value object which is not used anymore.

It's a bit different from usual Rails code as you're not using model objects (like AR models) to trigger updates, instead we're using command objects (like what gets called in `update_issue`) to trigger updates. More in the sense of the CQRS pattern.

If that's still not clear, let's discuss it and see how to make this more explicit in the code!